README.rst

```diff
@@ -15,23 +15,28 @@ Google Cloud Python Client
 .. _API Documentation: https://googlecloudplatform.github.io/google-cloud-python/stable/
 .. _Read The Docs Documentation: https://google-cloud-python.readthedocs.io/en/latest/
 
-This client supports the following Google Cloud Platform services:
-
--  `Google Cloud Datastore`_ (`Datastore README`_)
--  `Google Cloud Storage`_ (`Storage README`_)
--  `Google Cloud Pub/Sub`_ (`Pub/Sub README`_)
--  `Google BigQuery`_ (`BigQuery README`_)
--  `Google Cloud Resource Manager`_ (`Resource Manager README`_)
--  `Stackdriver Logging`_ (`Logging README`_)
--  `Stackdriver Monitoring`_ (`Monitoring README`_)
--  `Google Cloud Bigtable`_ (`Bigtable README`_)
--  `Google Cloud DNS`_ (`DNS README`_)
--  `Stackdriver Error Reporting`_ (`Error Reporting README`_)
--  `Google Cloud Natural Language`_ (`Natural Language README`_)
--  `Google Translate`_ (`Translate README`_)
--  `Google Cloud Vision`_ (`Vision README`_)
--  `Google Cloud Bigtable - HappyBase`_ (`HappyBase README`_)
--  `Google Cloud Runtime Configuration`_ (`Runtime Config README`_)
+
+This client supports the following Google Cloud Platform services at a Beta_ quality level:
+
+-  `Google BigQuery`_ (`BigQuery README`_) (Beta)
+-  `Google Cloud Datastore`_ (`Datastore README`_) (Beta)
+-  `Stackdriver Logging`_ (`Logging README`_) (Beta)
+-  `Google Cloud Storage`_ (`Storage README`_) (Beta)
```

Comment:
I was going off the screenshots for #2820. I thought that was strange as well.

@dhermes I'm not sure if they want these to match the other repos or not but maybe we could get a little more clarification from @omaray?